src/internal/xjwt_keyset.h

```diff
@@ -25,11 +25,11 @@ extern "C" {
 #endif /* __cplusplus */
 
 struct xjwt_keyset_t {
-  xjwt_key_t* keys;
+  xjwt_key_t *keys;
 };
 
-XJWT_API(xjwt_key_t*)
-xjwt_keyset__get_by_keyid(xjwt_keyset_t* ks, const char* keyid);
+XJWT_API(xjwt_key_t *)
+xjwt_keyset__get_by_keyid(struct xjwt_keyset_t *ks, const char *keyid);
```

Comment:
this should be type'defed so we don't need the struct bit here?

Reply:
I thought so too, but it was complaining about it. Does it work for you on OSX?